### PR TITLE
feat: simple yellow token

### PR DESCRIPTION
**Pull request by nksazonov**

### CODE REVIEW

File: contracts/SimpleYellowToken.sol [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	 * @dev Simple constructor, passing arguments to ERC20Permit and ERC20 constructors.
+	 * Mints the supply cap to the deployer.
+	 * @param name Name of the Token.
+	 * @param symbol Symbol of the Token.
+	 * @param supplyCap Maximum supply of the Token.
+	 */
+	constructor(
+		string memory name,
+		string memory symbol,
+		uint256 supplyCap
```
Comment (mod) [inline contracts/SimpleYellowToken.sol:24]: Name this supply or tokenSupply

File: contracts/SimpleYellowToken.sol [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import '@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol';
+
+/**
+ * @notice Yellow and Canary utility token is a simple ERC20 token with permit functionality.
+ * All the supply is minted to the deployer.
+ */
+contract SimpleYellowToken is ERC20Permit {
+	/// @dev Token maximum supply
+	uint256 public immutable TOKEN_SUPPLY_CAP;
```
Comment (mod) [inline contracts/SimpleYellowToken.sol:12]: There is no cap, this would imply we mint

File: contracts/SimpleYellowToken.sol [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	 * @return uint8 Number of decimals of Token.
+	 */
+	function decimals() public pure override returns (uint8) {
+		return 8;
+	}
+
+	/**
+	 * @notice Return the cap on the token's total supply.
+	 * @return uint256 Token supply cap.
+	 */
```
Comment (mod) [inline contracts/SimpleYellowToken.sol:42]: I would not include this method since cap = totalSupply

**Review by mod — CHANGES_REQUESTED** [review]

File: contracts/SimpleYellowToken.sol (hunk 1 above)
Comment (nksazonov) [reply contracts/SimpleYellowToken.sol]: ef857e48c551976dc091766f37676b89e005f14f

File: contracts/SimpleYellowToken.sol (hunk 2 above)
Comment (nksazonov) [reply contracts/SimpleYellowToken.sol]: ef857e48c551976dc091766f37676b89e005f14f

File: contracts/SimpleYellowToken.sol (hunk 3 above)
Comment (nksazonov) [reply contracts/SimpleYellowToken.sol]: ef857e48c551976dc091766f37676b89e005f14f

**Review by mod — APPROVED** [review]

**Pull request merged**